x/incentive/types/claims.go

```diff
@@ -167,12 +167,12 @@ type HardLiquidityProviderClaim struct {
 	BaseMultiClaim         `json:"base_claim" yaml:"base_claim"`
 	SupplyRewardIndexes    MultiRewardIndexes `json:"supply_reward_indexes" yaml:"supply_reward_indexes"`
 	BorrowRewardIndexes    MultiRewardIndexes `json:"borrow_reward_indexes" yaml:"borrow_reward_indexes"`
-	DelegatorRewardIndexes RewardIndexes      `json:"delegator_reward_indexes" yaml:"delegator_reward_indexes"`
+	DelegatorRewardIndexes MultiRewardIndexes `json:"delegator_reward_indexes" yaml:"delegator_reward_indexes"`
```

Comment:
I'm in support of
1. Breaking delegator rewards out of the Hard claim struct into a separate `DelegatorClaim` struct
2. Adding new msg `MsgClaimDelegatorRewards` to claim rewards stored by `DelegatorClaim`

But @rhuairahrighairidh and I discussed this a few days ago and decided against it based on the launch timeline (this was before we extended code freeze). This is currently a blocker for me because my [next task](https://app.asana.com/0/1156716699555540/1200318138217966/f) says "_...we combine the claim msgs, it should fulfill product requirements with minimal coding work. Though it does tradeoff some ease of use._"

If we decide to go this route we'll need to coordinate with API consumers (Cosmostation, etc.) as well since this would be a breaking change.